src/averageAges.js

```diff
@@ -15,6 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const mens = people.filter((person) => (person.sex === 'm'));
+  let newpeople = mens;
+
+  if (century) {
+    newpeople = mens.filter(
+      (person) => Math.ceil(person.died / 100) === century
+    );
+  }
+
+  const age = newpeople.map((persons) => persons.died - persons.born);
+
+  return (age.reduce((sum, x) => (sum + x), 0)) / age.length;
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
```

Comment:
remove comments

Reply:
why didn't you remove comments

Reply:
maybe you forgot to add your changes to git? please remove comments